src/components/DragAndDrop/Provider/index.js

```diff
@@ -36,6 +36,7 @@ function DragAndDropProvider({children, dropZoneID, dropZoneHostName, isDisabled
     const {windowWidth, windowHeight} = useWindowDimensions();
 
     const dropZone = useRef(null);
+    const dragCounter = useRef(0);
```

Comment:
If we drag on an element the counter will increase. If that element is removed from dom, the counter will not decrease (stale date). This caused https://github.com/Expensify/App/issues/34349